src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -345,20 +345,33 @@ void MergeLibraryManifest (string mergedManifest)
 			var nsResolver = new XmlNamespaceManager (new NameTable ());
 			nsResolver.AddNamespace ("android", androidNs.NamespaceName);
 			var xdoc = XDocument.Load (mergedManifest);
+			var package = xdoc.Root.Attribute ("package")?.Value ?? string.Empty;
+			log.LogDebugMessage ($"Resolved {package} from {mergedManifest}");
```

Comment:
1. Do we need this comment?
2. If we do, it should require less "context." For example, if `package` is the empty string, this will read:
    > Resolved  from path/to/mergedManifest.
    This would look *very* weird.